### PR TITLE
feat(editor3): added JIRA-like tooltips to Editor3

### PR DESCRIPTION
**Pull request by gbbr**
Uses `superdesk-ui-framework` tooltips.

### CODE REVIEW

**Comment by darconny** [comment]
@gbbr Can we use tooltips from UI framework?

**Comment by fritzSF** [comment]
If they work correctly for this we should definitely try to use the CSS only tooltips from the UI framework. @gbbr please try if they will work in this case. here is the page with examples: https://superdesk.github.io/superdesk-ui-framework/examples/

**Comment by gbbr** [comment]
Oh! Right. I didn't know about those. Unfortunately they don't work. React doesn't allow adding custom attributes, unless they are of the form `data-*`. So it would be great if the UI framework would allow both, in this case `sd-tooltip` _and_ `data-sd-tooltip`.

**Comment by darconny** [comment]
We added support for 'data-sd-tooltip' in UI framework, its merged.

**Comment by gbbr** [comment]
@darconny thanks! That's great! Do you think we can also add the `flow` attribute? In this case the tooltips need to show at the bottom, while here the default is at the top.

P.S. Ideally all ui-framework attributes should support the `data-` prefix to allow us to use them with React components, but for now, if you can just add `data-flow` that would be great.

**Comment by darconny** [comment]
@gbbr Now you can use 'data-flow' along with 'data-sd-tooltip'. Let me know if this works.

File: scripts/core/editor3/components/toolbar/StyleButton.jsx [hunk 1]
```diff
@@ -17,6 +17,22 @@ const StyleIcons = {
     orderedlist: 'icon-ordered-list'
 };
 
+const StyleTooltips = {
```
Comment (petrjasek) [inline scripts/core/editor3/components/toolbar/StyleButton.jsx]: would be good to use `gettext` for values

Comment (gbbr) [inline scripts/core/editor3/components/toolbar/StyleButton.jsx]: Damn! I keep forgetting this, sorry!

**Comment by gbbr** [comment]
@darconny Done with all changes. Thanks!

P.S. Wouldn't it be better to prefix all UI framework attributes the same? (`sd-*`). Why isn't flow prefixed I wonder? I'm sure it makes sense, I'm just curious...

**Comment by gbbr** [comment]
@darconny @fritzSF since you've asked for changes to this PR do you mind submitting a review and approving it, considering you are happy with the current state?

**Pull request merged**